pkgs/by-name/as/asfa/package.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  darwin,
+  gawk,
+  help2man,
+  openssl,
+  pkg-config,
+}:
+let
+  version = "0.10.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "asfa";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "obreitwi";
+    repo = "asfa";
+    rev = "v${version}";
+    hash = "sha256-MnhnwtZmPFhOuiqNiaxJnPu88JOdlpvyVy0YGphblBc=";
+  };
+
+  cargoHash = "sha256-/bRBP/NzcNOXl/nANeOYouUAo3NNbtbV9fxIJrNajYQ=";
+
+  buildInputs = [
+    openssl
+  ] ++ lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.SystemConfiguration ];
+
+  nativeBuildInputs = [
+    gawk
+    help2man
+    pkg-config
+  ];
+
+  doCheck = false;
+
+  postInstall = ''
+    mkdir -p "$out/man/man1"
+    help2man -o "$out/man/man1/asfa.1" "$out/bin/asfa"
+
+    # Generate info about all subcommands except for 'help' (which leads to error)
+    "$out/bin/asfa" --help | awk 'enabled && $1 != "help" { print $1 } /^SUBCOMMANDS:$/ { enabled=1 }' \
+        | while read -r cmd; do
+        help2man \
+            "--version-string=${version}" \
+            -o "$out/man/man1/asfa-$cmd.1" \
+            "$out/bin/asfa $cmd"
+    done
+  '';
+
+  meta = with lib; {
+    description = "Avoid sending file attachments by uploading them via SSH to a remote site and sharing a publicly-accessible URL with non-guessable (hash-based) prefix instead.";
```

IIRC `meta.description` shouldn't end with a period

```suggestion
    description = "Avoid sending file attachments by uploading them via SSH to a remote site and sharing a publicly-accessible URL with non-guessable (hash-based) prefix instead";
```

---

Statistics definitely support this → adapted
```
❯ grep -r 'description.*[^\.]";' pkgs | wc -l
48133
❯ grep -r 'description.*\.";' pkgs | wc -l
931
```